frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -13,7 +13,7 @@ export let AddDatasetDrawer = withStore('selectedDataset')(({ }) =>
     <h2 className='Secondary'>Upload your data to explore</h2>
     <p className='MediumWeight Muted'>
       To analyze the accessibility of your own set of providers, facilities, or social services, you
-      will need to upload two seperate CSV files:
+      will need to upload two separate CSV files:
       <ul>
         <li>list of service areas (County and/or ZIPCode columns).</li>
```

Comment:
Could we add the definitive list of required fields here?

Reply:
not in this PR!